packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1131,4 +1147,57 @@ contract Validators is
       }
     }
   }
+
+  /**
+   * @notice Sets the slashingMultiplierRestPeriod property if called by owner.
+   * @param value New reset period for slashing multiplier.
+   */
+  function setSlashingMultiplierResetPeriod(uint256 value) public nonReentrant onlyOwner {
+    slashingPenaltyResetPeriod = value;
+  }
+
+  /**
+   * @notice Resets a group's slashing multiplier if it has been >= the reset period since 
+   *         the last time the group was slashed.
+   */
+  function resetSlashingMultiplier() external nonReentrant {
+    address account = getAccounts().validatorSignerToAccount(msg.sender);
+    require(isValidatorGroup(account));
+    ValidatorGroup storage group = groups[account];
+    require(
+      now >= group.slashInfo.lastSlashed.add(slashingPenaltyResetPeriod),
+      "`resetSlashingMultiplier` called before resetPeriod expired"
+    );
+    group.slashInfo.penalty = FixidityLib.fixed1();
+  }
+
+  bytes32[] canHalveSlashingMultiplier = [
```

Comment:
@yorhodes suggested reading the `slasher` whitelist from `LockedGold` instead. Probably something to save for a future PR, since `isSlasher` doesn't exist yet